metaflow/mflog/__init__.py

```diff
@@ -43,13 +48,13 @@
 def bash_capture_logs(bash_expr, var_transform=None):
```

Comment:
I think it is worth renaming at least the argument, as it cannot be an arbitrary bash expression anymore. Also.. I'm pretty sure there is a bug here (and in the original PR) related to this, in cases when `bash_expr` is not a simple cmd + args

Reply:
See https://github.com/Netflix/metaflow/pull/805 for a tentative fix

Reply:
I can rename the arg. Keeping the function name constant will make it easier for @romain-intel to merge this PR into the converged branch.